signalfx-org-metrics/metrics.yaml

```diff
@@ -1457,6 +1457,164 @@ sf.org.limit.metricTimeSeriesCreatedPerMinute:
+ 
+sf.org.log.numContentBytesReceived:
+  brief: The volume of bytes Splunk Log Observer receives from ingesting logs off the wire after filtering and throttling. This content can be compressed.
+  description: |
+   The volume of bytes Splunk Log Observer receives from ingesting logs off the wire after filtering and throttling. This content can be compressed.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.numContentBytesReceived
+ 
+sf.org.log.grossMessagesReceived:
+  brief: The total number of log messages Splunk Log Observer receives before filtering and throttling.
+  description: |
+   The total number of log messages Splunk Log Observer receives before filtering and throttling.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.grossMessagesReceived
+ 
+sf.org.log.numMessageBytesReceived:
+  brief: The number of bytes Splunk Log Observer receives from ingested logs after decompression, filtering, and throttling are complete.
+  description: |
+   The number of bytes Splunk Log Observer receives from ingested logs after decompression, filtering, and throttling are complete.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.numMessageBytesReceived
+ 
+sf.org.log.grossContentBytesReceived:
+  brief: The volume of bytes Splunk Log Observer receives from ingesting logs off the wire before filtering and throttling. This content can be compressed.
+  description: |
+   The volume of bytes Splunk Log Observer receives from ingesting logs off the wire before filtering and throttling. This content can be compressed.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.grossContentBytesReceived
+ 
+sf.org.log.numMessagesReceivedByToken:
+  brief: The total number of log messages Splunk Log Observer accepts for a specific access token after filtering and throttling.
+  description: |
+   The total number of log messages Splunk Log Observer accepts for a specific access token after filtering and throttling.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.numMessagesReceivedByToken
+ 
+sf.org.log.numMessagesDroppedOversize:
+  brief: The number of log messages Splunk Log Observer receives that are too large to process.
+  description: |
+   The number of log messages Splunk Log Observer receives that are too large to process.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.numMessagesDroppedOversize
+ 
+sf.org.log.numMessagesDroppedThrottle:
+  brief: The number of log messages Splunk Log Observer drops after the allowed ingest volume is exceeded. Splunk Log Observer drops messages it receives after the ingestion volume is reached.
```

Comment:
Which ingest volume?  System wide, license allowable, etc?

Reply:
This is known definition. allowed ingest volume is always per org unless you have suffix of "ByToken".